components/styled/bookshelf.ts

```diff
@@ -5,21 +5,42 @@ const Container = tw.div`
   grid
```

Comment:
Bdw feel free to move these `styles` into its own file. Especially if its bloating the main component

Reply:
copy that. will do